debian/50-eos.preset

```diff
@@ -58,4 +58,5 @@ disable bootlogs.service
 # Units masked by other packages. Run the following to find them:
 #
 # find /usr/lib/systemd/system -lname /dev/null -printf 'disable %f\n' | sort
+disable eos-firewall-localonly.service
```

Comment:
Unless I missed it in the `eos-boot-helper` packaging, this unit is not masked (no `/etc/systemd/system/eos-firewall-localonly.service -> /dev/null` symlink). Please move this entry to the `# Disable other unwanted units` section.